Protocol.md

```diff
@@ -23,188 +44,182 @@ The following figure gives a brief overview of the protocol:
+  - include an adaptor signature in `cet_signatures` for every event specified in the `offer_dlc`'s `contract_info`.
+  - set `refund_signature` to the valid signature, using its `funding_pubkey` for the refund transaction, as defined in the [transaction specification](Transactions.md#refund-transaction).
+  - set `funding_signatures` to valid input signatures.
+  - include a signature in `funding_signatures` for every funding input specified in the`offer_dlc` message.
 
-### Accept validation
+The recipient:
 
-#### User validation
-* B fund amount: confirm that the fund amount matches what was agreed upon.
+  - if any `signature` is incorrect:
+    - MUST reject the contract.
+  - MUST NOT broadcast the funding transaction before receipt of a valid `sign_dlc`.
+  - on receipt of a valid `sign_dlc`:
+    - SHOULD broadcast the funding transaction.
 
-#### Machine validation
-* Bob public key: validate that the public key is well formed.
-* Bob inputs: validate that the referenced outputs are in fact unspent and are either P2WSH or P2WPKH.
-* CET signatures: validate that the signatures are valid for each of the CETs.
-* Refund signature: validate that the signature is valid.
-
-### Sign validation
-
-#### Machine validation
-* Fund transaction signatures: validate that the signatures are valid for each of the provided inputs.
-* CET signatures: validate that the signatures are valid for each of the CETs input.
-* Refund signature: validate that the signature is valid for the refund transaction input.
-
-## Contract cancellation
-
-At any point in the protocol up until the sending of the `sign` message, both parties can safely decide to cancel the contract.
-An empty `cancel` message can be used to inform the other party of the desire to cancel the contract.
-If one of the parties doesn't send a message after a given timeout, the other party should assume the contract to be canceled.
-Timeouts should be proportional to the number of possible outputs of the contract, as the signing and verification of the CET signatures is expected to be the bottleneck.
+# Authors
 
-After party A sends the `sign` message, they cannot unilaterally cancel the contract anymore.
-If they want to cancel the contract, for example because of not observing the fund transaction in the Bitcoin network, they can try to do so by broadcasting a transaction spending at least one of the UTXOs specified in the fund transaction.
-If this transaction gets included in the blockchain prior to the fund transaction, the contract will be canceled as this will have rendered the fund transaction invalid.
-In the event that the fund transaction gets included first, both parties will be bound to the contract.
+Nadav Kohen <nadavk25@gmail.com>
 
-# Authors
-Takatoshi Nakagawa
-Ichiro Kuwahara
-Thibaut Le Guilly
-Nadav Kohen
+[ FIXME: Add Authors ]
```

Comment:
still need?

Reply:
Yeah, all the BOLTs have this so I figured in this case I would put it here as unlike the tx doc others have worked on this pretty extensively previously